docs/machine-learning/how-to-guides/machine-learning-model-predictions-ml-net.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "How-To: Use a trained model to make predictions"
+description: Learn to make predictions using a trained model
+ms.date: 04/29/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc
+ms.topic: how-to
+#Customer intent: As a developer I want to use my model to make predictions
+---
+
+# How-To: Make predictions with a trained model
+
+Learn how to use a trained model to make predictions
+
+## Data Models
+
+### Input Data
+
+```csharp
+public class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+
+    [LoadColumn(1, 3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+
+    [LoadColumn(4)]
+    [ColumnName("Label")]
+    public float CurrentPrice { get; set; }
+}
+```
+
+### Output Data
+
+Like input column names, ML.NET has default names for columns that contain the predicted value of a model. Depending on the task it may be different. 
+
+Because the algorithm used in this sample is a linear regression algorithm, the default name of the output column is `Score` which is defined by the [`ColumnName`](xref:Microsoft.ML.Data.ColumnNameAttribute) attribute on the `PredictedPrice` property.
+
+```csharp
+class HousingPrediction : HousingData
+{
+    [ColumnName("Score")]
+    public float PredictedPrice { get; set; }
+}
+```
+
+The `HousingPrediction` data model inherits from `HousingData` to make it easy to visualize the original input data along with the output generated by the model.  
+
+## Set up Prediction Pipeline
+
+Whether making a single or batch prediction, the prediction pipeline needs to be set up. After loading the data preparation pipeline and the trained model, they're  combined into an [`ITransformerChain`](xref:Microsoft.ML.Data.TransformerChain%601).
+
+```csharp
+//Create MLContext 
+MLContext mlContext = new MLContext();
+
+// Load Data Preparation Pipeline and Trained Model
+DataViewSchema dataPrepPipelineSchema, modelSchema;
+ITransformer dataPreparationPipeline = mlContext.Model.Load("data_preparation_pipeline.zip", out modelSchema);
+ITransformer trainedModel = mlContext.Model.Load("model.zip", out modelSchema);
```

Comment:
idk if it would sound simpler if it was only one model saved, that contained the data prep and everything

Reply:
#resolved